tidy3d/components/simulation.py

```diff
@@ -812,6 +991,51 @@ def intersecting_media(
             mediums.update(_mediums)
         return mediums
 
+    @staticmethod
+    def intersecting_structures(
+        test_object: Box, structures: Tuple[Structure, ...]
+    ) -> Tuple[Structure, ...]:
+        """From a given list of structures, returns a list of :class:`Structure` that intersect
```

Comment:
let's use `.` before the class name inside of the :class: links. seems more robust.

Reply:
and fix below and other instances.